frontend/cypress/integration/tests/auth_spec.js

```diff
@@ -0,0 +1,51 @@
+const backend = 'localhost:3001';
+const testUsername = 'test';
+const testPassword = 'test';
+// Must be running the auth server for all tests.
```

Comment:
This doesn't need to be a Cypress test. Cypress is used to test frontend elements like clicking on pages or components, typing in fields etc. What you wrote here should be a Jest test instead.

Reply:
You could add a cypress test for logging in and registering, just use cy.get and cy.click to interact with frontend elements